File: Starter Problems 2/taktak.py
op = open('taktakin.txt', 'r')
f = int(op.readline().strip())
m = 0
while f % 11 != 1:
	f *= 2
	m += 1

open('taktakout.txt', 'w').write(str(m) + ' ' + str(f))

File: Starter Problems 2/taktak.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ifstream inFile;
    inFile.open("taktakin.txt");
    int fruits;
    inFile >> fruits;
    int months = 0;
    while (fruits % 11 != 1) {
        months ++;
        fruits *= 2;
    }
    inFile.close();
    ofstream outFile;
    outFile.open("taktakout.txt");
    outFile << months << " " << fruits;
    outFile.close();
}
